docs/_docs/installation/macos.md

```diff
@@ -3,155 +3,28 @@ title: Jekyll on macOS
 permalink: /docs/installation/macos/
 ---
 
-## Install Command Line Tools
-To install the command line tools to compile native extensions, open a terminal and run:
+## Supported macOS versions
 
-```sh
-xcode-select --install
-```
+- Monterey (macOS 12)
+- Big Sur (macOS 11)
+- Catalina (macOS 10.15)
```

Comment:
These are the OSes officially supported by Homebrew, and since we're using Homebrew to install a version manager, we have to mirror what Homebrew supports.